.lgtm.yml

```diff
@@ -0,0 +1,7 @@
+extraction:
+  cpp:
+    after_prepare:
+      - export GNU_MAKE=make
+    index:
+      build_command:
+        - $GNU_MAKE -j2 -s
```

Comment:
`ninja` is used on travis CI, I wonder if there's a way to use it in LGTM (would need to be installed).

Reply:
is `ninja` available?